auth_brute_force/models/res_authentication_attempt.py

```diff
@@ -34,29 +37,52 @@ class ResAuthenticationAttempt(models.Model):
     whitelisted = fields.Boolean(
         compute="_compute_whitelisted",
     )
+    banned = fields.Boolean(
```

Comment:
This field is confusing: we already have `result`, which says if it was banned; and we already have `whitelisted`, which says if the remote is whitelisted.

So IMHO you can remove this.

Reply:
This field shows the current state of the relation login/remote. It is used in order to show the option to unban a login/remote.

Reply:
That's the case for whitelisted field also.

Reply:
Mmh, no, because it will not be shown if you are currently unbanned.
Maybe, you are banned, once you are unbanned, you should not be unbanned again.

Reply:
True, but [if you don't create a new record](https://github.com/OCA/server-auth/pull/23/files#r191369394), then you'd be getting `result=unbanned`, which can then be used to apply [the domain to show or not the button](https://github.com/OCA/server-auth/pull/23/files#diff-d66823c3bc47d3ae32e26197db149004R31).